software/control/core/core.py

```diff
@@ -152,6 +161,10 @@ def set_crop(self, crop_width, crop_height):
         self.crop_width = crop_width
         self.crop_height = crop_height
 
+    def scale_crop(self, binning_x, binning_y):
+        self.crop_width = int(self.crop_width / binning_x)
+        self.crop_height = int(self.crop_height / binning_y)
```

Comment:
Need to be more defensive than this

Reply:
switching from 2x2 binning to 1x1 binning will cause the crop_width and crop_height to be wrong